.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Unless a substantial amount of contributors use VSCode, I'm a bit wary of adding this file to the repo – it's another thing to maintain even if infrequently used, and will be "noise" to non-VSCode users.

Reply:
I think almost everyone uses VSCode. Everyone in neherlab uses vscode - and you do too?

But sure, this was just a neat thing that I came across when trying to link to the extension.

Reply:
I do, but I can't speak for others in @blab or @nextstrain as a whole. Just wanted to point this out because I noticed that [no other @nextstrain repos have `.vscode/` checked in](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Anextstrain+path%3A.vscode) and I haven't noticed it much among other open source projects.

Consider this non-blocking though, unless anyone else has a strong opinion here.

Reply:
I don't use VSCode and tend to think editor configs should live outside the codebase, but I'm also fine with committing it if folks on the team will benefit and it's not high churn.

Reply:
I hear you - don't think there will be high churn, this is just to help set up snakefmt for now.

This has 0 impact on most folks since it's just "recommendation".